src/omp/exec_dist.f90

```diff
@@ -61,7 +61,7 @@ subroutine exec_dist_tds_compact( &
   end subroutine exec_dist_tds_compact
 
   subroutine exec_dist_transeq_compact( &
```

Comment:
Maybe add a comment explaining that rhs_du is where the results are also stored.